crates/valence_protocol/src/sound.rs

```diff
@@ -0,0 +1,32 @@
+// sound.rs exposes constant values provided by the build script.
+// All sounds are located in `Sound`. You can use the
+// associated const fn functions of `Sound` to access details about a sound.
+include!(concat!(env!("OUT_DIR"), "/sound.rs"));
+
+use crate::packets::s2c::play::SoundId;
+use crate::Ident;
+
+impl Sound {
+    pub fn to_id(self) -> SoundId<'static> {
+        SoundId::Direct {
+            id: Ident::new(self.to_str()).unwrap(),
```

Comment:
When `ident_str` is added in #32 it can be used here to avoid parsing the ident at runtime.